### ISSUE TITLE
A Channel in focus

### ISSUE THREAD

**Issue by jonaslagoni** [asyncapi/EDAVisualiser#13]
#### Reason/Context
So far all the views have been revolving around applications and how they interact with the world through channels.

But sometimes you wish to focus on a single channel and who interacts with that.

Therefore I am proposing we add a new view, similar to the `ApplicationFocusView`.

![image](https://user-images.githubusercontent.com/13396189/204195107-59e55d8c-5110-4b8b-93d1-a4a936d4e7c6.png)

But instead of having the application in the center, it will be a single channel with external applications around it, showing who receives from it and who sends to it.

**Comment by fmvilas** [comment]
Yeah, I'd say this is one of the most powerful use cases. This would help people make sense of their EDA.

Most of the time, the biggest concern people find about EDA and why they land on AsyncAPI, is that they have no idea who's consuming from or producing to a channel. They fear that if they change something on the message that's being published there, a consumer will break. And the result is that they end up creating a new channel so they don't break anything. This bloats their system with lots of unnecessarily duplicated (or almost duplicated) channels, making it more expensive, harder to maintain, and more energy-consuming.

So yeah, this view is really helpful.
